### DISCUSSION TITLE
GitHub Education Benefits not available yet

### DISCUSSION THREAD

**Discussion by Rian-Freitas** [community/community#143220] [GitHub Education]
### Hi everyone,

My request for the GitHub Education Benefits was approved October 29, 2024. However, I have not received my access to GitHub Copilot yet and I am wondering what happened. Is it okay and I should wait a little bit more or should I provide any other information?

**Comment by davevad93** [comment]
Hi @Rian-Freitas ,

Don't worry you don't have to do anything, the waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @Rian-Freitas was approved and your benefits are now available.`

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
